docs/_guide/templates.md

```diff
@@ -210,7 +232,7 @@ _my-element.js_
 
 Since the introduction of shadow DOM, we use the term "light DOM" to refer to nodes that appear in the main DOM tree.
```

Comment:
It's a little odd that this sentence appears before the reference to light DOM in the text in the next sentence. Merge?

Reply:
It is a weird sentence. If we're going to use the term "light DOM," we should define it. (Otherwise, maybe we should just say, "children"?)

Reply:
If there's only one use of the term "light DOM" in the doc, might be worth rewording to avoid introducing the terminology. Or maybe define it after the usage?

Reply:
Removing the light DOM references.